app/ts/McdaApp/Workspace/CurrentTab/Preferences/EquivalentChange/EquivalentChangeUtil.test.ts

```diff
@@ -73,68 +73,58 @@ describe('tradeOffUtil', () => {
     });
   });
 
-  describe('getImprovedValue', () => {
-    const usePercentage = false;
-    const criterionWeight = 0.1;
-    const partOfInterval = 0.5;
-    const referenceWeight = 0.5;
-
+  const usePercentage = false;
+  const criterionWeight = 0.1;
+  const partOfInterval = 0.5;
+  const referenceWeight = 0.4;
+  describe('getEquivalentRangeValue', () => {
     it('should return an improved value given a worst value, considering the weights, and ratio to the reference criterion, for an increasing pvf', () => {
       const pvf = {range: [0, 1], direction: 'increasing'} as TPvf;
-      const result = getImprovedValue(
+      const result = getEquivalentRangeValue(
         usePercentage,
         criterionWeight,
         pvf,
         partOfInterval,
         referenceWeight
       );
-      expect(result).toEqual(2.5);
+      expect(result).toEqual(2);
     });
 
     it('should return an improved value given a worst value, considering the weights, and ratio to the reference criterion, for a decreasing pvf', () => {
       const pvf = {range: [0, 1], direction: 'decreasing'} as TPvf;
-      const result = getImprovedValue(
+      const result = getEquivalentRangeValue(
         usePercentage,
         criterionWeight,
         pvf,
         partOfInterval,
         referenceWeight
       );
-      expect(result).toEqual(-1.5);
+      expect(result).toEqual(-1);
     });
   });
 
-  describe('isImprovedValueRealistic', () => {
-    const usePercentage = false;
-    it('should return true if the improved value is below the best possible value for the criterion given an increasing pvf', () => {
-      const value = 0.5;
-      const pvf = {direction: 'increasing', range: [0, 1]} as TPvf;
-      const result = isImprovedValueRealistic(value, usePercentage, pvf);
-      expect(result).toBeTruthy();
-    });
-
-    it('should return false if the improved value is above the best possible value for the criterion given an increasing pvf', () => {
-      const value = 1.5;
-      const usePercentage = false;
-      const pvf = {direction: 'increasing', range: [0, 1]} as TPvf;
-      const result = isImprovedValueRealistic(value, usePercentage, pvf);
-      expect(result).toBeFalsy();
-    });
-
-    it('should return true if the improved value is above the best possible value for the criterion given an decreasing pvf', () => {
-      const value = 0.5;
-      const usePercentage = false;
-      const pvf = {direction: 'decreasing', range: [0, 1]} as TPvf;
-      const result = isImprovedValueRealistic(value, usePercentage, pvf);
-      expect(result).toBeTruthy();
+  describe('getEquivalentValue', () => {
+    it('should return the equivalent change for a second criterion, based on its pvf, a reference change and the relative weights of the criteria, for increasing pvf', () => {
+      const pvf = {range: [0, 1], direction: 'increasing'} as TPvf;
+      const result = getEquivalentValue(
+        usePercentage,
+        criterionWeight, // 0.1
```

Comment:
remove comments or leave for clarification?

Reply:
leave in